src/javascript/app/base/header.js

```diff
@@ -361,23 +373,26 @@ const Header = (() => {
-                    ? localizeKeepPlaceholders('Your account has not been authenticated. Please submit your [_1]proof of identity and proof of address[_2] to authenticate your account and request for withdrawals.', 'user/authenticate')
-                    : localizeKeepPlaceholders('Your account has not been authenticated. Please submit your [_1]proof of identity and proof of address[_2] to authenticate your account and access your cashier.'), 'user/authenticate'),
+                cashier_locked    : () => localize('Your cashier is currently locked. Please contact us via live chat to find out how to unlock it.'),
+                system_maintenance: () => getSystemMaintenanceMessage(),
+                currency          : () => buildMessage(localizeKeepPlaceholders('Please set your [_1]account currency[_2] to enable deposits and withdrawals.'),                                                                                    'user/set-currency'),
+                unsubmitted       : () => buildMessage(get_account_status.risk_classification === 'high'
+                    ? localizeKeepPlaceholders('Your account has not been authenticated. Please submit your [_1]proof of identity, proof of address and proof of income[_2] to authenticate your account and request for withdrawals.', 'user/authenticate')
+                    : localizeKeepPlaceholders('Your account has not been authenticated. Please submit your [_1]proof of identity, proof of address and proof of income[_2] to authenticate your account and access your cashier.'), 'user/authenticate'),
                 expired                   : () => localize('The identification documents you submitted have expired. Please submit valid identity documents to unlock Cashier.'),
-                expired_identity          : () => buildMessage(localizeKeepPlaceholders('Your [_1]proof of identity[_2] has expired.'),'user/authenticate'),
-                expired_document          : () => buildMessage(localizeKeepPlaceholders('Your [_1]proof of address[_2] has expired.'),'user/authenticate', '?authentication_tab=poa'),
+                expired_identity          : () => buildMessage(localizeKeepPlaceholders('Your [_1]proof of identity[_2] has expired.'),                                                                                         'user/authenticate'),
+                expired_document          : () => buildMessage(localizeKeepPlaceholders('Your [_1]proof of address[_2] has expired.'),                                                                                          'user/authenticate', '?authentication_tab=poa'),
+                expired_income            : () => buildMessage(localizeKeepPlaceholders('Your [_1]proof of income[_2] has expired.'),                                                                                          'user/authenticate', '?authentication_tab=edd'),
```

Comment:
check please this, as I know we don’t have 'expired' option for proof of income feature.
as I know, being verified it has no expiring period